app/src/events/position_state_changed.c

```diff
@@ -7,4 +7,15 @@
 #include <kernel.h>
 #include <zmk/events/position_state_changed.h>
 
-ZMK_EVENT_IMPL(zmk_position_state_changed);
\ No newline at end of file
+ZMK_EVENT_IMPL(zmk_position_state_changed);
+
+uint32_t zmk_last_event_trace_id = 0;
```

Comment:
These should be static unless you mean for them to be globals accessible from other files (without going through zmk_event_trace_id).